modules/entrezdirect/esearch/meta.yml

```diff
@@ -0,0 +1,51 @@
+name: "entrezdirect_esearch"
+description: Searches a term in a public NCBI database
+keywords:
+  - public datasets
+tools:
+  - entrezdirect:
+      description: |
+        Entrez Direct (EDirect) is a method for accessing the NCBI's set of
+        interconnected databases (publication, sequence, structure, gene,
+        variation, expression, etc.) from a UNIX terminal window. Functions
+        take search terms from command line arguments. Individual operations
+        are combined to build multi-step queries. Record retrieval and
+        formatting normally complete the process.
+      homepage: https://www.ncbi.nlm.nih.gov/books/NBK179288/
+      documentation: https://www.ncbi.nlm.nih.gov/books/NBK25501/
+      tool_dev_url: https://www.ncbi.nlm.nih.gov/books/NBK25498/
+      doi: 10.1016/S0076-6879(96)66012-1
+      licence: ["PUBLIC DOMAIN"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - database:
+      type: value
+      description: Value must be a valid Entrez database name (default=pubmed).
```

Comment:
It won't be in this case, as you sitll supply `-db` (I assume it will error out if there isn't actually a string available for the parameter)?

Reply:
Yes, in fact there is no default, so it will be better to delete that part:

```
  - database:
      type: value
      description: Value must be a valid Entrez database name.
```